### PR TITLE
Add TradingView icon

### LINKED ISSUE #8805: Add TradingView icon
### Brand Name

TradingView

### Website

https://www.tradingview.com/

### Popularity Metric

The SimilarWeb rank is 284. See https://www.similarweb.com/website/tradingview.com/#overview.

### Official Resources for Icon and Color

Brand overview from Pentagram (creator): https://www.pentagram.com/work/tradingview
SVG: https://tradingview.com/static/images/svg/tradingview-with-text-logo.svg
SVG code for symbol only (from tradingview.com heading):
```
<svg width="36" height="28" viewBox="0 0 36 28" xmlns="http://www.w3.org/2000/svg">
<path d="M14 22H7V11H0V4h14v18zM28 22h-8l7.5-18h8L28 22z" fill="currentColor"/>
<circle cx="20" cy="8" r="4" fill="currentColor"/>
</svg>
```

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by bneeland**
**Issue:** closes #8805

**Similarweb rank:**
284
https://www.similarweb.com/website/tradingview.com/#overview

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Adds TradingView icon.

tradingview

### CODE REVIEW

**Review by LitoMore — CHANGES_REQUESTED** [review]
image

@bneeland I used https://tradingview.com/static/images/svg/tradingview-with-text-logo.svg as reference but found some differences.

Please make sure you're using the correct resource as reference.

File: icons/tradingview.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9.465 18.193H4.732v-7.57H0V5.808h9.465v12.386Zm9.465 0h-5.41l5.07-12.386H24l-5.07 12.386Zm-5.41-6.881c1.494 0 2.705-1.232 2.705-2.753 0-1.52-1.21-2.752-2.704-2.752-1.494 0-2.705 1.232-2.705 2.752 0 1.52 1.211 2.753 2.705 2.753Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/tradingview.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9 6H0v4.5h4.5V18H9V6zm10.5 12h-6L18 6h6l-4.5 12zm-6.75-7.5c1.2426 0 2.25-1.0074 2.25-2.25S13.9926 6 12.75 6 10.5 7.0074 10.5 8.25s1.0074 2.25 2.25 2.25z"/></svg>
```
Took SVG from their website header and resized it to project specifications.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Fixed the SVG by taking it directly from the brand website

**Review by LitoMore — DISMISSED** [review]
I use this one below as a reference:

```svg
<svg width="36" height="28" viewBox="0 0 36 28" xmlns="http://www.w3.org/2000/svg">
	<path d="M14 22H7V11H0V4h14v18zM28 22h-8l7.5-18h8L28 22z" fill="currentColor"/>
	<circle cx="20" cy="8" r="4" fill="currentColor"/>
</svg>
```

But still found some differences:

image

File: icons/tradingview.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9 6H0v4.5h4.5V18H9V6zm10.5 12h-6L18 6h6l-4.5 12zm-6.75-7.5c1.2426 0 2.25-1.0074 2.25-2.25S13.9926 6 12.75 6 10.5 7.0074 10.5 8.25s1.0074 2.25 2.25 2.25z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/tradingview.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9.465 18.085H4.732v-7.437H0V5.915h9.465v12.17zm9.465 0h-5.41l5.07-12.17H24l-5.07 12.17zm-5.41-12.17a2.704 2.704 0 1 1 0 5.409 2.704 2.704 0 0 1 0-5.409z"/></svg>
```
This should fix the differences. Perplexed as to why the previous suggestion didn't work

**Review by PratyushJayachandran — DISMISSED** [review]
Please merge my suggestion.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Even with @PratyushJayachandran's changes - I'm seeing [major differences](https://github.com/simple-icons/simple-icons/assets/1830867/f7bcf3b2-e905-4b19-88b2-52182219bacd) when compared to the SVG linked by @LitoMore. This'll still need some work I'm afraid!

File: _data/simple-icons.json [hunk 3]
```diff
@@ -12122,6 +12122,11 @@
             "hex": "FFC107",
             "source": "https://github.com/tqdm/img/blob/0dd23d9336af67976f88f9988ea660cde78c54d4/logo.svg"
         },
+        {
+            "title": "TradingView",
+            "hex": "131622",
+            "source": "https://tradingview.com"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:12128]:
```suggestion
            "source": "https://www.tradingview.com/media-kit"
```

File: icons/tradingview.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9.465 18.085H4.732v-7.437H0V5.915h9.465v12.17zm9.465 0h-5.41l5.07-12.17H24l-5.07 12.17zm-5.41-12.17a2.704 2.704 0 1 1 0 5.409 2.704 2.704 0 0 1 0-5.409z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/tradingview.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M15.8654 8.2789c0 1.3541-1.0978 2.4519-2.452 2.4519-1.354 0-2.4519-1.0978-2.4519-2.452 0-1.354 1.0978-2.4518 2.452-2.4518 1.3541 0 2.4519 1.0977 2.4519 2.4519zM9.75 6H0v4.9038h4.8462v7.2692H9.75Zm8.5962 0H24l-5.1058 12.173h-5.6538z"/></svg>
```

**Review by PeterShaggyNoble — DISMISSED** [review]

**Comment by PeterShaggyNoble** [comment]
In the interests of getting this merged in, I've taken another pass at this. This latest version is based on `TradingView.zip\TradingView-media-kit\logo\svg\black\icon-logo.svg` from the media kit available at https://www.tradingview.com/media-kit/ - from their own usage I think we're OK to leave the background off and just use the icon.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**